docs/blueprints/EP031.rst

```diff
@@ -292,7 +323,15 @@ The **telemetry_int** napp must use a different cookie ID to help understanding
 XI. Consistency
 ===============
 
-The **telemetry_int** napp will deploy a routine to evaluate the consistency of the telemetry flows as performed by the **mef_eline** napp. This implementation will be defined via field experience with Kytos. The consistency check will rely on ``sdntrace_cp`` and follow the same pattern as ``mef_eline``, except that also when trying to trace, it should test both UDP and TCP payloads, if any fails after a few attempts, then it should disable telemetry int and remove the flows for now, falling back to mef_eline flows. In the future, the consistency check process might evolve, but for now if it fails, it will fail safely falling back to mef_eline flows. As of ``sdntrace_cp`` version ``2023.1`` it still doesn't completely support ``goto_table`` neither ``instructions``, so it needs to be augmented just so ``telemetry_int`` can eventually also rely on it. 
+The **telemetry_int** napp will deploy a routine to evaluate the consistency of the telemetry flows as performed by the **mef_eline** napp. This implementation will be defined via field experience with Kytos. The consistency check will rely on ``sdntrace_cp`` and follow the same pattern as ``mef_eline``, except that also when trying to trace, it should test both UDP and TCP payloads, if any fails after a few attempts, then it should disable telemetry int and remove the flows for now, falling back to mef_eline flows. In the future, the consistency check process might evolve, but for now if it fails, it will fail safely falling back to mef_eline flows.
+
+The consistency check will be implemented after version ``2024.1`` when **mef_eline** implements its enhanced consistency check, and when it's been battle tested for some time, which is expected to also check for active EVCs. **mef_eline** enhanced consistency check details will be specified in a new blueprint. But, in general, it's expected to:
```

Comment:
Initially, it was being discussed whether or not `telemetry_int` consistency check would be shipped on `2024.1`, but since it depends on `mef_eline` indirectly, let's sure make sure `mef_eline` enhanced consistency is battle tested. Currently, in prod the `mef_eline` consistency check isn't activating circuits either anymore in practice, so, in the meantime external scripts can be used dispatching some sdntrace_cp traces covering UDP and TCP payload if Ops might want to. Since there's this related dependency here: I'm trying to minimize changes that would need to be refactored while still shipping something with quality. Let me know if you guys have any other preference or recommendation.